daml-script/test/BUILD.bazel

```diff
@@ -169,7 +170,21 @@ EOF
 
 da_scala_library(
     name = "test-utils",
-    srcs = glob(["src/test-utils/**/*.scala"]),
+    srcs = glob(["src/test-utils/**/*.scala"]) +
+    #  we depend on sources to avoid pushing a canton artifact to maven
+    [
+        "//canton:community_base_proto_scala",
+    ],
+    override_scalacopts = [
+        copt
+        for copt in default_scalacopts + lf_scalacopts_stricter
+        if copt not in [
+            "-P:wartremover:traverser:org.wartremover.warts.NonUnitStatements",
+        ]
+    ] + [
+        # ParticipantTransactionProto uses a deprecated message, silence the warning.
+        "-Wconf:cat=deprecation&src=com/digitalasset/canton/protocol/v30/ParticipantTransactionProto.scala:s",
+    ],
```

Comment:
Given this is a test, you can probably directly depend on the base_proto bazel rule from canton/BUILD, rather than depending on the source here. We only did this to avoid bloating the daml-script jar too much

Reply:
done :)